ruby/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ruby:3.0
```

Comment:
Use ENV variable and use it in your code

Reply:
Done